rocAL/rocAL/include/augmentations/geometry_augmentations/node_resize_mirror_normalize.h

```diff
@@ -25,26 +25,29 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_vx.h"
 
-class ResizeMirrorNormalizeNode : public Node
-{
+class ResizeMirrorNormalizeNode : public Node {
 public:
     ResizeMirrorNormalizeNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     ResizeMirrorNormalizeNode() = delete;
-    void init(std::vector<float>& mean,  std::vector<float>& std_dev, IntParam *mirror);
+    void init(unsigned dest_width, unsigned dest_height, RocalResizeScalingMode scaling_mode, std::vector<unsigned> max_size,
+              RocalResizeInterpolationType interpolation_type,std::vector<float>& mean,  std::vector<float>& std_dev, IntParam *mirror);
```

Comment:
Space after comman is to be maintained

RocalResizeInterpolationType interpolation_type, std::vector<float>& mean